app/assets/javascripts/init.coffee

```diff
@@ -114,6 +114,16 @@ class BikeIndex.Init extends BikeIndex
       return unless time.isValid
       $this.text(displayLocalDate(time, $this.hasClass("preciseTime")))
 
+    # Write local time (format: 1569955709)
+    $(".convertUnixTime").each ->
```

Comment:
Rather than doing this, the `convertTime` function in this file should be updated to match the function in [time_parser.js](https://github.com/bikeindex/bike_index/blob/master/app/javascript/packs/utils/time_parser.js) - which converts both timestamps and `ISO_8601`.

Sorry, should've pointed to this earlier

Reply:
Can you move this fix into a separate PR?

Reply:
Thanks, updated -- https://github.com/bikeindex/bike_index/pull/1294